vanilla_js/server.js

```diff
@@ -0,0 +1,277 @@
+// Express server previewing how to best interact with common Pelm API endpoints
```

Comment:
all suggestions from other `server.js` file should be applied here

Reply:
mirrored!